package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-materialize",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
Thanks! but don't worry about the versioning, we'll handle that when publishing :)